packages/editor/src/components/collapsible-block-toolbar/style.scss

```diff
@@ -77,3 +77,6 @@
 	}
 }
 
+.editor-header__toolbar:has(.editor-collapsible-block-toolbar:not(.is-collapsed)) + .editor-header__center {
+	display: none;
+}
```

Comment:
I’d put this in the parent component’s styles instead because this doesn’t apply to any elements of the component itself.